docs/TROUBLESHOOTING.md

```
@@ -36,18 +36,14 @@ If you need help, first search if there is [already an issue filed](https://issu
     oc logs -f deploy/velero -n openshift-adp
     ```
     
-1. Velero commands
-    -  Alias the velero command: 
-    ```
-    alias velero='oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero'
-    ```
+1. OADP CLI commands
     - Get the backup details: 
     ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)